popup/popup.js

```diff
@@ -49,28 +56,68 @@ function listenForClicks() {
     });
 }
 
-async function darkModeSwitch(){
-    if(window.darkMode){
-        await browser.tabs.removeCSS({file:"./../dark-mode/dark-mode.css"})
-    }else{
-        await browser.tabs.insertCSS({file:"./../dark-mode/dark-mode.css"})
+function sendMessageToTabs(tabs, data) {
+    /**
+     * Send Message to the content-script
+     */
+    for (let tab of tabs) {
+        browser.tabs.sendMessage(
+            tab.id,
+            data
+        ).then(response => {
+            // alert(JSON.stringify(response));
+            if (response.DarkMode == "Off") {
+                window.darkMode = false;
+            }
+            else {
+                window.darkMode = true;

```

Comment:
window.darkMode = response.DarkMode !== "Off"

Reply:
I don't quite understand this comment, but just to clarify a bit.
The `window.darkMode` "static variable" is necessary for the `darkModeSwitch()` function below, and I don't have an access to the `response` in there. (the response is from the content script, namely `storageLoader.js`)

Reply:
Yeah I understand, just wrote a shortcut for the if-else statement

Reply:
Oh I see I'll change it in a new commit, thanks!